From 880a3468f0ab61d51a5b91290a3a4c38be5693cc Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Tue, 28 Jul 2026 15:35:22 +0800
Subject: [PATCH 434/441] AOSCOS: init: allow disabling DEBUG_KERNEL when
 EXPERT is selected

This made no sense in our case, especially for Afterglow.

Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 init/Kconfig | 2 --
 1 file changed, 2 deletions(-)

diff --git a/init/Kconfig b/init/Kconfig
index abd4845b7fbe..57f01b204ded 100644
--- a/init/Kconfig
+++ b/init/Kconfig
@@ -1695,8 +1695,6 @@ config HAVE_PCSPKR_PLATFORM
 
 menuconfig EXPERT
 	bool "Configure standard kernel features (expert users)"
-	# Unhide debug options, to make the on-by-default options visible
-	select DEBUG_KERNEL
 	help
 	  This option allows certain base kernel options and settings
 	  to be disabled or tweaked. This is for specialized
-- 
2.55.0

